requirements.txt

```diff
@@ -20,3 +20,6 @@ mypy
 pytest >= 4.6
 pytest-cov
 pytest-runner
+chemicals
+thermo
+pyvista
```

Comment:
chemicals is need when loading chemical species and creating fluid components
thermo is needed later, because the Peng-Robinson EoS relies on its database for binary interaction parameters between components (Though that one I could eliminate by telling the user to provide the BIPs)
pyvista is probably an artifact from collaboration with @OmarDuran

Reply:
Removing thermo and pyvista.

@keileg  Do you want to keep chemicals?
The ``load_species`` function tries to import the module locally, so we can turn it into a utility function where the user has to ensure the package is there (similar to what Peter did for the progress bar with tqdm).
Users can create an instance of Species anyways by providing required values.